packages/editor/src/browser/editor-preferences.ts

```diff
@@ -13,7 +13,7 @@ import {
     PreferenceContribution,
     PreferenceSchema,
     PreferenceChangeEvent
-} from '@theia/preferences-api';
+} from '@theia/core/lib/browser/preferences-api';
```

Comment:
let's use `@theia/core/lib/browser/preferences` instead `preferences-api` and please make sure that preferences are reexported by `@theia/core/lib/browser`